sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-footer.ts

```diff
@@ -25,15 +25,14 @@ export class ECertPartTimeFileFooter extends ECertFileFooter {
 
   createFromLine(line: string): ECertPartTimeFileFooter {
     const footer = new ECertPartTimeFileFooter();
-    footer.recordTypeCode = line.substring(0, 3) as RecordTypeCodes;
-    // Here total record count is the total records.
-    footer.recordCount =
-      parseInt(line.substring(43, 52)) + parseInt(line.substring(52, 61));
-    footer.totalSINHash = parseInt(line.substring(61, 76));
+    footer.recordTypeCode = line.substring(0, 2) as RecordTypeCodes;
+    // Total number of records not including header and footer.
+    footer.recordCount = +line.substring(51, 60);
```

Comment:
Is line.substring(43, 52) not included here is about not including header and footer.

Reply:
Not really, I was not able to understand what was the idea from the previous implementation.
`line.substring(43, 52)` represents the total of records that were successfully processed. The successfully processed records are not part of the file so they should not be part of the total records validation.
![image](https://user-images.githubusercontent.com/61259237/233717587-838ccb13-2435-4a38-81f9-e3403f02f35c.png)

Reply:
I see that is weird. We can ask the reason to the person who is dealing with the file integration if not already asked.

Reply:
I asked them for a sample file and I did the changes based on the file provided (that is aligned with the documentation), so it should be ok.
What is weird to me is how it worked or was even tested before, maybe some misdirection from previous documentation.
Either way, the upcoming tests will have the files validated.